testsuite/features/support/twopence_init.rb

```diff
@@ -98,7 +98,8 @@ def twopence_init(target)
   $rhlike_minion = twopence_init("ssh:#{ENV['RHLIKE_MINION']}") if ENV['RHLIKE_MINION']
   $deblike_minion = twopence_init("ssh:#{ENV['DEBLIKE_MINION']}") if ENV['DEBLIKE_MINION']
   $build_host = twopence_init("ssh:#{ENV['BUILD_HOST']}") if ENV['BUILD_HOST']
-  $nodes += [$minion, $ssh_minion, $rhlike_minion, $deblike_minion, $build_host]
+  $sles_salt_migration = twopence_init('ssh:salt-sles-migration-test.tf.local')
```

Comment:
There is no environment variable in the `.bashrc` on the controller. I did not find it necessary to add it there since we only use this nested VM for the Salt bundle migration tests.

Reply:
Fixed with adjustments to sumaform. It works the same way as the others above.